src/test/javascript/spec/service/quiz-exercise.service.spec.ts

```diff
@@ -133,7 +133,7 @@ describe('QuizExercise Service', () => {
         expect(req.request.url).toEndWith(urlSuffix);
         req.flush(response);
         const resp = await result;
-        expect(resp.ok);
+        expect(resp.ok).toBe(true);
```

Comment:
```suggestion
        expect(resp.ok).toBeTrue();
```

Reply:
Once [eslint-plugin-jest-extended](https://github.com/jest-community/eslint-plugin-jest-extended) is available, we will also automatically flag this. 🙂